src/Microsoft.Kiota.Serialization.Form.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageProjectUrl>https://aka.ms/kiota/docs</PackageProjectUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <Deterministic>true</Deterministic>
-    <VersionPrefix>1.1.4</VersionPrefix>
+    <VersionPrefix>1.1.5</VersionPrefix>
```

Comment:
can we also bump the required abstractions version to ensure no misalignment happens on the client side please?

Reply:
Can do, I was going to wait until it's released to bump, but I guess we already know what the version number will be